dashboard/modules/serve/tests/test_serve_dashboard.py

```diff
@@ -742,5 +743,54 @@ def test_target_capacity_field(ray_start_stop, url: str):
     assert requests.put(url, json=config, timeout=30).status_code == 400
 
 
+def check_log_file(log_file: str, expected_regex: list):
+    with open(log_file, "r") as f:
+        s = f.read()
+        for regex in expected_regex:
+            assert re.findall(regex, s) != []
+    return True
+
+
+def test_put_with_logging_config(ray_start_stop):
+    """Test serve component logging config can be updated via REST API."""
+
+    url = "http://localhost:8265/api/serve/applications/"
+    import_path = "ray.serve.tests.test_config_files.logging_config_test.model"
+    config1 = {
+        "http_options": {
+            "host": "127.0.0.1",
+            "port": 8000,
+        },
+        "logging_config": {
+            "encoding": "JSON",
+        },
+        "applications": [
+            {
+                "name": "app",
+                "route_prefix": "/app",
+                "import_path": import_path,
+            },
+        ],
+    }
+    # Ensure the REST API is idempotent
+    num_iterations = 2
+    for iteration in range(num_iterations):
```

Comment:
Why do we need multiple iterations here? This test seems to deploy an identical config each time?

Reply:
looks like it is for testing idempotent for deploying same config multiple times. i follow our style in our test case.

Reply:
The other tests check idempotence by applying a new config or deleting a config in each iteration. I don't think we're testing anything important by reapplying the same config.

Reply:
got it, removed.